homepage.css

```diff
@@ -20,55 +24,57 @@ body{
 
 nav{
 	width:100%;
-	background:white;
+	background-color:#0C59CF;
 	overflow:auto;
+	
 }
 
 ul{
 	padding:0;
-	margin:0 0 0 150px;
+	margin:0 0 0 30px;
```

Comment:
26/8/2021: I change the margin so it right for my link because the old margin was wrong and make me have to reposition of the drop down menu column